test/e2e/clients.go

```diff
@@ -7,6 +7,7 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/client/config"
 
+
```

Comment:
Seems there is a bug in sort-imports. And the test fails, because fmt is removing this line again